README.md

```diff
@@ -1,6 +1,9 @@
 # nsd_hed_labels
 This repository contains [Hierarchical Event Descriptors (HED)](https://www.hedtags.org/) labels of 1015 images from the [Natural Scenes Dataset (NSD)](https://naturalscenesdataset.org/) images. This is work in progress...
 
+## Read the Docs 
+[https://nsd-hed-labels-rtd-test.readthedocs.io/en/latest/](https://nsd-hed-labels-rtd-test.readthedocs.io/en/latest/)
```

Comment:
Change to MNL reathedocs version: https://nsd-hed-labels.readthedocs.io/en/latest/